packages/stan/src/test/derived.js

```diff
@@ -0,0 +1,169 @@
+/**
+ * Internal dependencies
+ */
+import { createAtomRegistry, createAtom, createDerivedAtom } from '../';
+
+async function flushImmediatesAndTicks( count = 1 ) {
+	for ( let i = 0; i < count; i++ ) {
+		await jest.runAllTicks();
+		await jest.runAllImmediates();
+	}
+}
+
+jest.useFakeTimers();
+
+describe( 'creating derived atoms', () => {
+	it( 'should allow creating derived atom', async () => {
+		const count1 = createAtom( 1 );
+		const count2 = createAtom( 1 );
+		const count3 = createAtom( 1 );
+		const sum = createDerivedAtom(
+			( { get } ) => get( count1 ) + get( count2 ) + get( count3 )
+		);
+		const registry = createAtomRegistry();
+
+		// Atoms don't compute any value unless there's a subscriber.
+		const unsubscribe = registry.subscribe( sum, () => {} );
+		expect( registry.get( sum ) ).toEqual( 3 );
+		registry.set( count1, 2 );
+		expect( registry.get( sum ) ).toEqual( 4 );
+		unsubscribe();
```

Comment:
Ideally, `unsubscribe` would be called in `afterEach` section when set by the test to avoid issues when a test failure impacts other tests. 

Although, it might be non issue here as registries seem to be isolated.